syntax = "proto2";
package psyche;
option optimize_for = LITE_RUNTIME;

message PingRequest {
  required int32 token = 1;
}

message PingResponse {
  required int32 token = 1;
}

service PsycheDemoServer {
  rpc Ping (PingRequest) returns (PingResponse);
}
